src/main/java/com/prgrms/needit/domain/member/dto/MemberCreateRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.prgrms.needit.domain.member.dto;
+
+import com.prgrms.needit.domain.member.entity.Member;
+import javax.validation.constraints.NotBlank;
+import lombok.Getter;
+
+@Getter
+public class MemberCreateRequest {
+
+	@NotBlank
+	private String email;
+	@NotBlank
+	private String nickname;
+	@NotBlank
+	private String password;
+	@NotBlank
+	private String address;
+	@NotBlank
+	private String contact;
+	@NotBlank
+	private String profileImageUrl;
```

Comment:
```suggestion
	@NotBlank
	private String email;
	
	@NotBlank
	private String nickname;
	
	@NotBlank
	private String password;
	
	@NotBlank
	private String address;
	
	@NotBlank
	private String contact;
	
	@NotBlank
	private String profileImageUrl;
```
필드마다 개행을 주면 깔끔하고 보기 쉬울 것 같아용!!